pycyphal/transport/can/media/pythoncan/_pythoncan.py

```diff
@@ -254,6 +259,29 @@ def configure_acceptance_filters(self, configuration: typing.Sequence[FilterConf
         _logger.debug("%s: Acceptance filters activated: %s", self, ", ".join(map(str, configuration)))
         self._bus.set_filters(filters)
 
+    def transmit_thread_worker(self) -> None:
+        try:
+            while not self._closed:
+                tx_tuple = self._tx_queue.get(block=True)
+                try:
+                    if self._closed or tx_tuple is None:
+                        return
+
+                    message: can.Message = tx_tuple[0]
+                    timeout: float = tx_tuple[1]
+                    future: asyncio.Future[None] = tx_tuple[2]
+                    loop: asyncio.AbstractEventLoop = tx_tuple[3]
+                    self._bus.send(message, timeout)
+                    loop.call_soon_threadsafe(lambda: future.set_result(None))
+                except Exception as ex:
+                    loop.call_soon_threadsafe(lambda: future.set_exception(ex))
```

Comment:
The future may be undefined here because it comes from within the try block. Mypy should have raised an error about this! Please always run it locally and configure your IDE to perform type-checking in real time. I would also add a debug-level log message here about the error in case the recipient thread is dead.